build.sh

```diff
@@ -86,6 +87,19 @@ if [[ $1 == "hypervisor-standalone" ]]; then
     echo "You have the docker-compose.hypervisor-standalone.yml and docker-compose.hypervisor-standalone.devel.yml compose files. Have fun!"
 fi
 
+if [[ $1 == "video-standalone" ]]; then
+    echo "Building video-standalone.yml..."
```

Comment:
File name incorrect, but we can merge it and fix it solving #248.
```suggestion
    echo "Building docker-compose.video-standalone.yml..."
```